source/_patterns/02-organisms/content-headers/content-header.mustache

```diff
@@ -121,13 +121,15 @@
       <ol class="content-header__author_list">
         {{#authors.list}}
           <li class="content-header__author_list_item">
-            <span class="content-header__author">{{#url}}<a href="{{url}}" data-behaviour="Popup" class="content-header__author_link">{{/url}}{{name}}{{#url}}</a>{{/url}}{{#isCorresponding}}
-              <picture>
+            <span class="content-header__author">{{#url}}<a href="{{url}}" data-behaviour="Popup" class="content-header__author_link">{{/url}}{{name}}{{#url}}</a>{{/url}}{{#isCorresponding}}&nbsp;{{!
```

Comment:
Can still recreate the original problem, looks like `&nbsp;` doesn't fix non-text?

Reply:
https://stackoverflow.com/questions/25041857/why-do-inline-blocks-break-after-non-breaking-space